tools/webpack/blocks.js

```diff
@@ -135,23 +135,26 @@ module.exports = [
 						{
 							from: `${ from }/**/*.php`,
 							to( { absoluteFilename } ) {
-								const [ , dirname, basename ] =
+								const [ , dirname, _basename ] =
```

Comment:
This change was necessary to prevent an eslint error regarding the scope of the `basename` variable.

Reply:
I'd prefer to completely rename one of these basenames, maybe this one should become `fileBasename`? The `_` prefix often indicates unused or private variables and I think this can be more descriptive.

Reply:
Certainly. I updated it that way.